source/utilities.tex

```diff
@@ -17022,6 +17122,69 @@
 the same type, if any, as does \tcode{common_type_t<T2, T1>}.
 No diagnostic is required for a violation of this Note's rules.
 
+\pnum
+For the \tcode{common_reference} trait applied to a parameter pack \tcode{T} of
+types, the member \tcode{type} shall be either defined or not present as follows:
+\begin{itemize}
+\item If \tcode{sizeof...(T)} is zero, there shall be no member \tcode{type}.
+
+\item Otherwise, if \tcode{sizeof...(T)} is one, let \tcode{T0} denote the sole
+  type in the pack \tcode{T}. The member typedef \tcode{type} shall denote the
+  same type as \tcode{T0}.
+
+\item Otherwise, if \tcode{sizeof...(T)} is two, let \tcode{T1} and \tcode{T2}
+  denote the two types in the pack \tcode{T}. Then
+  \begin{itemize}
+  \item If \tcode{T1} and \tcode{T2} are reference types and
+    \tcode{COMMON_REF(T1, T2)} is well-formed, then the member typedef
+    \tcode{type} denotes that type.
+
+  \item Otherwise, if \tcode{basic_common_reference<remove_cvref_t<T1>, remove_cvref_t<T2>,
+    \brk{}XREF(\brk{}T1), XREF(T2)>::type} is well-formed, then the member typedef
+    \tcode{type} denotes that type.
+
+  \item Otherwise, if \tcode{COND_RES(T1, T2)} is well-formed, then the
+    member typedef \tcode{type} denotes that type.
+
+  \item Otherwise, if \tcode{common_type_t<T1, T2>} is well-formed, then the
+    member typedef \tcode{type} denotes that type.
+
+  \item Otherwise, there shall be no member \tcode{type}.
+  \end{itemize}
+
+\item Otherwise, if \tcode{sizeof...(T)} is greater than two, let \tcode{T1},
+  \tcode{T2}, and \tcode{Rest}, respectively, denote the first, second, and
+  (pack of) remaining types comprising \tcode{T}. Let \tcode{C} be the type
+  \tcode{common_reference_t<T1, T2>}. Then:
+  \begin{itemize}
+  \item If there is such a type \tcode{C}, the member typedef \tcode{type} shall
+    denote the same type, if any, as \tcode{common_reference_t<C, Rest...>}.
+
+  \item Otherwise, there shall be no member \tcode{type}.
+  \end{itemize}
+\end{itemize}
+
+\pnum
+Notwithstanding the provisions of\iref{meta.type.synop}, and
+pursuant to\iref{namespace.std}, a program may specialize
```

Comment:
Both \iref's should be \ref's (with a space in front); the \iref automatically adds parens that we don't want here (compare to the parallel phrasing for common_type two paragraphs above).

Reply:
I somewhat blindingly replaced `ref`s with `iref`s - I'll audit them all.